ui/app/src/components/ContextMenu.tsx

```diff
@@ -73,38 +69,62 @@ export default function ContextMenu(props: CustomMenuProps) {
       open={open}
       classes={{ paper: classes?.paper, list: classes?.menuList }}
       onClose={onClose}
+      anchorOrigin={anchorOrigin}
     >
+      <div>
+        <ContextMenuItems
+          sections={sections}
+          classes={classes}
+          onMenuItemClicked={onMenuItemClicked}
+        />
+      </div>
```

Comment:
Is the wrapper div required?

Reply:
yes, because the issue with the foward ref